metecho/api/models.py

```diff
@@ -1047,7 +1058,9 @@ def queue_refresh_org(self, *, originating_user_id):
         self.has_been_visited = False
         self.currently_refreshing_org = True
         self.save()
-        self.notify_changed(originating_user_id=originating_user_id)
+        self.notify_changed(
+            type_="SCRATCH_ORG_RECREATE", originating_user_id=originating_user_id,
+        )
```

Comment:
@wlonk I don't think we want this to change -- it seems unrelated to the rest of this PR. From the perspective of the front end, "refreshing" an org is not the same as undeleting an org.